src/MinedMap.cpp

```diff
@@ -192,10 +192,13 @@ static int64_t getModTime(const std::string &file) {
 		return INT64_MIN;
 	}
 
-#ifdef _WIN32
+#if defined(_WIN32)
 	return (int64_t)s.st_mtime * 1000000;
+#elif defined(__APPLE__)
+    // stat.h syntax has changed on MacOSX SDK since at least 10.15
```

Comment:
Are you sure that this has changed at all? As far as I know, MinedMap never worked on OSX. Maybe we can add this to the commit message, but I don't think it needs a code comment.

Reply:
I mean, I'm not positive, but I assume that at some point the Apple, generic unix, and other linux libraries had the same syntax.